src/librustc_typeck/check/mod.rs

```diff
@@ -3762,6 +3794,13 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                 ),
                 DiagnosticId::Error(error_code.to_owned()),
             );
+            let label = format!("supplied {}", potentially_plural_count(arg_count, "parameter"));
```

Comment:
```suggestion
            let label = format!("supplied {}", potentially_plural_count(arg_count, "argument"));
```
And in comments as well.